appveyor.yml

```diff
@@ -123,6 +127,32 @@ branches:
   only:
     - auto
 
+after_test:
```

Comment:
I am concerned about this after_test stuff running on *all* AppVeyor build jobs rathern than only on those that are going to be "deployed".